plugin/firewall/firewall.go

```diff
@@ -103,12 +106,15 @@ func (p *firewall) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Ms
 		return dns.RcodeSuccess, nil
 	case policy.TypeBlock:
 		// One of the RuleList ended evaluation with typeBlock : return the initial request with corresponding rcode
+		log.Warning("coredns::policy/firewall, Action is Block")
```

Comment:
IIUC, these additional warnings would result in too much logging.

Reply:
changed the severity to debug